packages/providers/upload-aws-s3/README.md

```diff
@@ -25,6 +25,10 @@ npm install @strapi/provider-upload-aws-s3 --save
 
 - `provider` defines the name of the provider
 - `providerOptions` is passed down during the construction of the provider. (ex: `new AWS.S3(config)`). [Complete list of options](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#constructor-property)
+- `providerOptions.params` is passed directly to the parameters to each method respectively.
+  - `ACL` is the access control list for the object. Defaults to `public-read`.
```

Comment:
maybe to be super clear we should say "set ACL to private to use a private s3 bucket with the Strapi upload plugin"?

or maybe thats too much and any users using a private bucket will already know how this works. WDYT?

Reply:
I think if they have configured a private bucket they should know how ACL works. But maybe we could add what are the other options. I will add it in another PR 👍